src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -65,7 +64,6 @@ export interface TerminalResourceRequestConfig {
 	foldersRequested?: boolean;
 	cwd?: URI;
 	pathSeparator: string;
```

Comment:
I think this type should live in `extHost.protocol.ts` as that defines types used across exthost and main procs?

Reply:
can't access that from the browser AFAICT?

Reply:
extHost.protocol.ts is in common, should be fine?

Reply:
got an import warning when I tried to use it here.